<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicerConnectEditor</class>
 <widget class="QWidget" name="SlicerConnectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SlicerConnectEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="connectionCollapsible">
     <property name="text">
      <string>Connection Settings</string>
     </property>
     <layout class="QFormLayout" name="connectionLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="tokenLabelText">
        <property name="text">
         <string>Authentication:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="tokenLabel">
        <property name="text">
         <string>Token: Not Found</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="statusLabelText">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="refreshConnectionButton">
        <property name="text">
         <string>Refresh Connection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="qMRMLSegmentEditorWidget" name="editorWidget"/>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="statsCollapsible">
     <property name="text">
      <string>Statistics</string>
     </property>
     <layout class="QFormLayout" name="statsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sentLabelText">
        <property name="text">
         <string>Updates Sent:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="sentLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="receivedLabelText">
        <property name="text">
         <string>Updates Received:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="receivedLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="connectedUsersLabelText">
        <property name="text">
         <string>Connected Users:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="connectedUsersLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="logCollapsible">
     <property name="text">
      <string>Message Log</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="logLayout">
      <item>
       <widget class="QTextEdit" name="logTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

